Misc/NEWS.d/next/Library/2023-07-15-12-52-50.gh-issue-105726.NGthO8.rst

```diff
@@ -0,0 +1,3 @@
+Added ``__slots__`` to :class:`contextlib.AbstractContextManager` and :class:`context.AbstractAsyncContextManager`
+so that child classes can use ``__slots__``.
+specifying slots in the child-class does not lead to the immutability of child-class instance.
```

Comment:
```suggestion
Added ``__slots__`` to :class:`contextlib.AbstractContextManager` and :class:`context.AbstractAsyncContextManager`
so that child classes can use ``__slots__``.

```